chart/assets/operations/instance_groups/scheduler.yaml

```diff
@@ -88,10 +88,6 @@
 - type: remove
   path: /instance_groups/name=scheduler/jobs/name=cc_deployment_updater
 
-{{- if not .Values.features.eirini.enabled }}
```

Comment:
We still have a scheduler patch, and you're running it… I assume this means we now use the scheduler even in the Eirini case? I don't see any matching changes in this PR though?

Reply:
The code that removes the `/tasks` functionality from the `opi/task_client.rb` is for `cloud_controller_clock`, which is part of `scheduler`.

The other `scheduler` patch is removing the `KUBERNETES_SERVICE_HOST` environment variable, and it seems fine for the Eirini branch too, so I removed the conditional instead of hard-coding patch names in the scheduler template.